### DISCUSSION TITLE
Make conform load the .clang-format file

### DISCUSSION THREAD

**Discussion by Buer-Nahida** [LazyVim/LazyVim#1700] [Q&A] [answered]
Before latest version 10, I always use `none-ls` and .clang-format file to formatting .cpp file.
In latest version 10, `none-ls` is deprecated, so I tried migrating the `none-ls` configuration to `conform`, but `conform` plugin can't load .clang-format file.
This means that I will not be able to automatically format .cpp file using the project's .clang-format file.
I tryed use `lsp.none-ls` extra but if I enable `none-ls`, .lua file it's can't use auto formatting feature.
So can I make conform load .clang-format file to formatting .cpp file ?

**Comment by lcampit** [comment]
I have the same issue with prettier not using the project defined prettierrc to format files

**Reply by jcma86** [reply]
> I have the same issue with prettier not using the project defined prettierrc to format files

did you find a solution?

**Comment by Buer-Nahida** [comment, marked as answer]
Close, Because it's answered. Answer is
```diff
1   return {
2      "stevearc/conform.nvim",
3      dependencies = { "clangd_extensions.nvim", "mason.nvim" },
4      opts = function()
5          local opts = {
6              formatters_by_ft = {
- 7               c = { "clang-format" },
- 8               cpp = { "clang-format" },
- 9               objc = { "clang-format" },
- 10              objcpp = { "clang-format" },
- 11              cuda = { "clang-format" },
- 12              proto = { "clang-format" },
+ 7               c = { "clang_format" },
+ 8               cpp = { "clang_format" },
+ 9               objc = { "clang_format" },
+ 10              objcpp = { "clang_format" },
+ 11              cuda = { "clang_format" },
+ 12              proto = { "clang_format" },
13            },
14        }
15        return opts
16    end,
17 } 
```

**Reply by prabanjanraja** [reply]
tried this but not working
